docs/source/api/pjmedia/pjmedia-audiodev.rst

```diff
@@ -98,9 +98,12 @@ like this::
 
 .. _opensl:
 
-Android OpenSL
+Android OpenSL (deprecated)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Note that Android OpenSL ES has been deprecated in favor of :ref:`oboe`.
+
+
 .. _jnisound:
 
 Android JNI
```

Comment:
Perhaps we need to write something here (FYI currently this is the default backend for Android)?

The backend is actually accessing Android Java API [AudioTrack](https://developer.android.com/reference/android/media/AudioTrack) & [AudioRecord](https://developer.android.com/reference/android/media/AudioRecord) via JNI.

Also, perhaps some credits (not sure if it is relevant):
```
/* This file is the implementation of Android JNI audio device.
 * The original code was originally part of CSipSimple
 * (http://code.google.com/p/csipsimple/) and was kindly donated
 * by Regis Montoya.
 */
```

Reply:
it's very very relevant! Pls let me know if there other parts of the docs that need attributions